docs/testing.rst

```diff
@@ -313,7 +313,8 @@ And then to use it::
         return jsonify(username=g.user.username)
 
     with user_set(app, my_user):
-        with app.test_client() as c:
```

Comment:
```suggestion
        with app.test_client() as c:```suggestion
        with app.test_client() as c:
```

```

Reply:
@georgekald79 please stop reviewing random PRs in projects you aren'te ven involved with, especially when the reviews/suggestions are not actually useful.